operator/roles/ansible-service-broker/templates/broker-operator.monitor-rolebinding.yaml.j2

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
It's weird to me that `prometheus-k8s` doesn't have permission to view all of this stuff across the whole cluster.